src/django_components/component.py

```diff
@@ -422,12 +422,12 @@ def render(self, context: Context) -> str:
                         f"Detected duplicate fill tag name '{resolved_name}'."
                     )
 
-                resolved_fill_alias = fill_node.resolve_alias(context, resolved_component_name)
```

Comment:
I renamed the internal variables from `alias` to `slot_default_var` and from `scope` to `slot_data_var`, so it's clear what they refer to.